toolbox/fdc3-conformance/src/test/manual/fdc3.manual.ts

```diff
@@ -1,25 +1,25 @@
-import { DesktopAgent } from '@finos/fdc3';
+import { FDC3ChannelChangedEvent, getAgent } from '@finos/fdc3';
 import { closeMockAppWindow } from '../fdc3-2_0-utils';
 import { APIDocumentation2_0 } from '../support/apiDocuments-2.0';
 import { ContextType, IntentApp, Intent, RaiseIntentControl2_0 } from '../support/intent-support-2.0';
 import { handleFail, wait } from '../../utils';
+import { expect } from 'chai';
 
-const control = new RaiseIntentControl2_0();
 const raiseIntentDocs = '\r\nDocumentation: ' + APIDocumentation2_0.raiseIntent + '\r\nCause';
 
-declare let fdc3: DesktopAgent;
-
 /**
  * Details on the mock apps used in these tests can be found in /mock/README.md
  */
 
-export let fdc3ResolveAmbiguousIntentTarget_2_0 = () =>
-  describe('ResolveAmbiguousIntentTarget_2.0', () => {
+export const fdc3ResolveAmbiguousIntentTarget_2_0 = () =>
+  describe('2.0-ResolveAmbiguousIntentTarget', async () => {
```

Comment:
Maybe I am just confused but why the different approach to adding 2.0? Sometimes after, sometimes before?

Reply:
the text string is what appears on the screen.  This is consistent.  The constant is used in the code and is also consistent but just with the other constants :)